osbs/api.py

```diff
@@ -642,6 +646,8 @@ def create_prod_build(self, *args, **kwargs):
         :param outer_template: str, name of outer template for BuildRequest
         :param customize_conf: str, name of customization config for BuildRequest
         :param arrangement_version: int, numbered arrangement of plugins for orchestration workflow
+        :param signing_intent: str, signing intent of the ODCS composes
```

Comment:
I'm wondering if adding these here will cause confusion since any odcs support in prod builds will be accidental :)

Reply:
I've just add it here as _do_prod doesn't have any docstring, so at least it will be together?

Reply:
Probably fine to leave it here then.

Reply:
ok